### ISSUE TITLE
How to do this!

### ISSUE THREAD

**Issue by Jhair2015** [Tyler3D/Multi-Ogar-Edited-Unlimited#101]
Well, I tried to make autestart as easy, I mean leaderboard is set to 'restart' when a few seconds are missing, @Tyler3D  could tell me if this is possible and how? please.

**Comment by Tyler3D** [comment]
@Jhair2015 Could you rephrase your question?

**Comment by Jhair2015** [comment]
@Tyler3D https://streamable.com/zi7bi

check out leaderboard

**Comment by Tyler3D** [comment]
@Jhair2015 If there is little people in the leaderboard the same effect should happen. The way it is coded now wouldn't allow for a complete override of the leaderboard

**Comment by Jhair2015** [comment]
If it happens, but I have eliminated the lines 67, 68 and 69
So I do not take all the leaderboard
I have tried to do that without affecting the 10 seconds before the autestarting in ...
